internal/testutil/mig/pre_check.go

```diff
@@ -29,6 +29,12 @@ func PreCheckCert(tb testing.TB) {
 	acc.PreCheckCert(tb)
 }
 
+func PreCheckLDAP(tb testing.TB) {
+	tb.Helper()
+	checkLastVersion(tb)
+	acc.PreCheckLDAP(tb)
```

Comment:
migration test do not need MONGODB_ATLAS_LDAP_CA_CERTIFICATE?

Reply:
the 2 tests that need are skipped